### ISSUE TITLE
Create a tickets purchase form.

### ISSUE THREAD

**Issue by marcoag** [fossasia/open-event-server#9107]
WARNING: This is an issue for the new version v3 of the eventyay backend API, make sure you are working with the [dev-v3 branch](https://github.com/fossasia/open-event-server/tree/dev-v3), and that your PRs target that specific branch.

Depends on #9106.

Create a form that allows for the selection and purchase of tickets. Have a look at how tickets are bought for other events in eventyay (ie. [fossasia 2023](https://eventyay.com/e/55d2a466)).

For the scope of this issue no payments need to be processed. All payments can be considered successful, we will take care of payments in another issue.

**Comment by kulsoomzahra** [comment]
I would like to work on submitting a PR for this issue.

**Comment by techmannih** [comment]
I m working on this issue

**Comment by hpdang** [comment]
Please note this issue can be taken by multiple participants. It will remain open and accessible to all participants until a successful PR is submitted.

**Comment by ErNehaSharma** [comment]
Hello sir/ma'am I want to work on this issue.

**Comment by Djpatel00** [comment]
Do you have any specific requirement or something in mind to make?

**Comment by sarveshwarikumari05** [comment]
Hi, I am new to the project and would like to work on this issue, I am setting up the dev-v3 branch locally and going through the related flow. Please let me know if I can proceed or if there are any pointers.
Thanks!
